plugin/cmd/kubectl/binding/binding.go

```diff
@@ -0,0 +1,86 @@
+/*
+Copyright 2016 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/kubernetes-incubator/service-catalog/plugin/pkg/kubectl/client"
+	"github.com/kubernetes-incubator/service-catalog/plugin/pkg/kubectl/utils"
+)
+
+const usage = `Usage:
+  kubectl plugin binding SUBCOMMAND
+
+Available subcommands:
+  list
+  get
+`
+
+const listUsage = `Usage:
+  kubectl plugin binding list NAMESPACE
+`
+
+const getUsage = `Usage:
+  kubectl plugin binding get NAMESPACE INSTANCENAME
```

Comment:
Minor thing, but INSTANCENAME` would be easier to read as `INSTANCE_NAME`. Same for other 2-word thingies.